plotly_resampler/figure_resampler/figure_resampler.py

```diff
@@ -41,26 +41,53 @@ def __init__(
         show_mean_aggregation_size: bool = True,
         convert_traces_kwargs: dict | None = None,
         verbose: bool = False,
+        show_dash_kwargs: dict | None = None,
```

Comment:
No docstring for this argument. Generated docs look like this:
![image](https://user-images.githubusercontent.com/38005924/178301221-d35759c3-1b5a-4f79-b85a-0fbda052dfef.png)

Reply:
fixed.